stable/redmine/templates/_helpers.tpl

```diff
@@ -154,3 +154,45 @@ Return the name of the Secret used to store the passwords
 {{ template "redmine.fullname" . }}
 {{- end -}}
 {{- end -}}
+
+
+
```

Comment:
please remove extra newlines

Reply:
I am sorry. I have not responded you (I thought I did).
I removed the newlines (and made some changes as requested by @juan131 ).